docs/_theme/section.hbs

```diff
@@ -142,4 +142,11 @@
     <div class='pad1y quiet space-top2 prose-big'>Events</div>
     <%= renderSectionList({ members: section.members.events, renderSection: renderSection, noun: 'Event' }) %>
   <% } %>
+
+  <% if (section.sees) { %>
+    <div class='pad1y quiet space-top2 prose-big'>See</div>
```

Comment:
I agree. We may also want to use `@see` tags to link to resources other than examples such as the style spec, TileJSON spec, GeoJSON spec, etc